src/main/resources/edu/cornell/kfs/module/ld/document/datadictionary/CuYearEndSalaryExpenseTransferDocument.xml

```diff
@@ -42,4 +57,14 @@
     </property>
   </bean>
 
+  <bean id="YearEndSalaryExpenseTransferDocument-workflowAttributes" parent="YearEndSalaryExpenseTransferDocument-workflowAttributes-parentBean">
```

Comment:
Making changes as requested.  Like you, I believe routing is valid as it currently stands as it is associated with parent document LaborDistributionYearEndTransactionalDocument.